engines/python/src/main/java/ai/djl/python/engine/PyProcess.java

```diff
@@ -122,6 +122,9 @@ synchronized void startPythonProcess() {
             int id = restartCount.get();
             int port = connections.get(0).getPort();
             logger.info("Start process: {} - retry: {}", port, id);
+            if (pyEnv.isEnableVenv()) {
```

Comment:
create/delete venv should happen at model loading and close time, not at StartPyProcess time

one model may have multiple workers.

Reply:
Changed.